EthereumKit/Networking/Client/Configuration.swift

```diff
@@ -16,6 +16,9 @@ public struct Configuration {
             
         case .ropsten:
             return URL(string: "https://ropsten.etherscan.io")!
+            
+        case .geth(_):
+            return URL(string: "https://ropsten.etherscan.io")!
         }
```

Comment:
the endpoint should be passed by the end user because it could be localhost

Reply:
I'll Add a note